slack_wrapper.cfg

```diff
@@ -1,2 +1 @@
-SLACK_URL="https://hooks.slack.com/services/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"
```

Comment:
As SLACK_URL is a secret and this repository is public i moved the SLACK_URL in the server (this one is invalid since long time)